src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueNoticeHandler.java

```diff
@@ -63,4 +63,8 @@ protected String buildSignedInJudgeName(String userAuthorisation) {
         return userDetailsService.buildLoggedInUserName(userAuthorisation);
     }
 
+    protected String buildSignedInJudgeSurname(String userAuthorisation) {
```

Comment:
Is this necessary given the method above it is doing the same thing? Could we instead simply rename the above to something more generic like buildSignedInJudgeName and pass our surname into it?

Reply:
As per the requirement only the surname is required to be displayed in the report.
I have added a separate new method to get only the surname of the logged in User. The exiting method return the full name i.e (first name + surname) and exiting method is used in some other places as well.  To minimise the impact of the changes added a new method return only the surname and also i have add one method **getUserDetails** which can be used for generic conditions.

Reply:
Right I think I understand, then shouldn't this method be calling userDetailsService.buildLoggedInUserSurname() instead of buildLoggedInUserName()?

Otherwise it is functionally the same.

Reply:
Thanks for the comments.
Modified as per the review comment and committed the changes..